### ISSUE TITLE
Attempt to invoke virtual method 'int java.lang.String.indexOf(java.lang.String)' on a null object reference

### ISSUE THREAD

**Issue by liccowee** [coomar2841/image-chooser-library#166]
![img_0228](https://cloud.githubusercontent.com/assets/921872/14875229/357d4118-0d3b-11e6-9015-8b6344beacd9.JPG)

Some of the photo in my google photos does not seem to able for picking. Anyone have this similar issue?

**Comment by coomar2841** [comment]
Are you on the latest version?

**Comment by liccowee** [comment]
Yes. I pretty sure it is latest version. I tried download the official latest app from google store and it still the same error. Any clue?
